pkg/execution/execution_test.go

```diff
@@ -345,16 +346,16 @@ func TestExecutionInfo(t *testing.T) {
 		var exec = require('k6/x/execution');
 
 		exports.default = function() {
-			var ti = exec.test;
-			if (ti.duration !== 0) throw new Error('unexpected test duration: '+ti.duration);
+			var ti = exec.instance;
+			if (ti.currentTestRunDuration !== 0) throw new Error('unexpected test duration: '+ti.currentTestRunDuration);
 			if (ti.vusActive !== 1) throw new Error('unexpected vusActive: '+ti.vusActive);
-			if (ti.vusMax !== 0) throw new Error('unexpected vusMax: '+ti.vusMax);
+			if (ti.vusInitialized !== 0) throw new Error('unexpected vusInitialized: '+ti.vusInitialized);
 			if (ti.iterationsCompleted !== 0) throw new Error('unexpected iterationsCompleted: '+ti.iterationsCompleted);
 			if (ti.iterationsInterrupted !== 0) throw new Error('unexpected iterationsInterrupted: '+ti.iterationsInterrupted);
 		}`},
 		{name: "test_err", script: `
 		var exec = require('k6/x/execution');
-		exec.test;
+		exec.instance;
 		`, expErr: "getting test information in the init context is not supported"},
```

Comment:
```suggestion
		`, expErr: "getting instance information in the init context is not supported"},
```

Reply:
hmm :+1: but I also have to fix the original error message, not just the test

Reply:
Resolved by https://github.com/grafana/xk6-execution/pull/4/commits/29813c13c5091e1745ce2af5b2a86eac2b6bb364